blog/css/style.css

```diff
@@ -0,0 +1,36 @@
+.container {
+	padding-top: 15px;
+	max-width: 1000px;
+    margin-left: auto;
+	margin-right: auto;
+}
+
+#blog_post_section .entry-row {
+	box-sizing: border-box;
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.blog_post {
+	text-align: center;
+	width: 100%;
+}
+
+.entry-box {
+	text-align: center;
+}
+
+.entry-row {
+	padding-top: 10px;
+	padding-bottom: 10px;
+	width: 100%;
+}
+
+/*.entry-row>label {
```

Comment:
You shouldn't commit commented out code generally unless you have a specific reason that is also listed in the comment.